manifests/mongos/config.pp

```diff
@@ -51,24 +53,4 @@
     group   => 'root',
     mode    => '0644',
   }
-
-  if $service_manage {
-    if $facts['os']['family'] == 'RedHat' or $facts['os']['family'] == 'Suse' {
-      file { '/etc/sysconfig/mongos' :
```

Comment:
This file is not mentioned in the service unit.
The content is set as `Environment` in the unit.